packages/networks/src/networks/gnosis.ts

```diff
@@ -42,6 +42,7 @@ export const gnosis: NetworkConfig = {
     coinbase: 'DAI',
   },
   startBlock: 19338700,
+  swapPurchaser: '0x72381052e4F7765A00a403891420BF75876c75bB',
```

Comment:
Oh! is that v3?

Reply:
no... actually uniswap is not deployed on gnosis either :/ Like bsc, it was triggered by the "deploy all" script